erpnext/accounts/report/inactive_sales_items/inactive_sales_items.py

```diff
@@ -96,6 +96,8 @@ def get_data(filters):
 						"qty": item_obj.qty,
 						"days_since_last_order": item_obj.days_since_last_order
 					})
+				else:
+					continue
```

Comment:
Instead of this move data.append(row) in the if condition next to 98 line. But this looks like intentionally kept outside of the If condition.

Reply:
Yes it is intentional, if the item is not there in sales_invoice_date that means that item has never been sold in the respective territory. So the row should also be appended.